src/en/mangago/src/eu/kanade/tachiyomi/extension/en/mangago/Mangago.kt

```diff
@@ -151,11 +162,21 @@ class Mangago : ParsedHttpSource() {
 
     override fun searchMangaNextPageSelector() = genreListingNextPageSelector
 
+    private val titleRegex = Regex("""\(yaoi\)|\{Official\}|«Official»|〘Official〙|\(Official\)|\s\[Official]|\s「Official」|『Official』|\s?/Official\b""", RegexOption.IGNORE_CASE)
+    private fun titleVersion(title: String) = title.replace(titleRegex, "").substringBeforeLast("(").trim()
```

Comment:
is the substringbeforelast necessary?

Reply:
not really... I just tested it out after removing it and it worked after all 
mb